tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts

```diff
@@ -0,0 +1,684 @@
+
+    const docType = new DocumentTypeBuilder()
+      .withName(documentName)
+      .withAlias(documentAlias)
+      .withAllowAsRoot(true)
+      .addGroup()
+        .withName('BlockListGroup')
+        .addCustomProperty(dataType['id'])
+          .withAlias(blockListAlias)
+        .done()
+      .done()
+      .addGroup()
+        .withName('TheBlockListGroupTheSecond')
+        .addCustomProperty(dataType['id'])
+          .withAlias('theBlockListAliasTheSecond')
+        .done()
+      .done()
+      .build();
+    const generatedDocType = await umbracoApi.documentTypes.save(docType);
+
+    const rootContentNode = new ContentBuilder()
+      .withContentTypeAlias(generatedDocType["alias"])
+      .withAction("saveNew")
+      .addVariant()
+        .withName(blockListName)
+        .withSave(true)
+        .addProperty()
+          .withAlias(blockListAlias)
+          .addBlockListValue()
+            .addBlockListEntry()
+              .withContentTypeKey(element['key'])
+              .appendContentProperties(element.groups[0].properties[0].alias, "aliasTest")
+            .done()
+          .done()
+        .done()
+      .done()
+      .build();
+    await umbracoApi.content.save(rootContentNode);
+
+    await umbracoUi.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.refreshContentTree();
+
+    // Opens the content with the block list editor
+    await page.locator('[data-element="tree-item-' + blockListName + '"]').click();
+
+    // Checks to make sure that there is only one item in the first group
+    await expect(page.locator('[data-element="group-aBlockListGroup"] >> .umb-block-list__block--view')).toHaveCount(1);
+    // Checks to make sure that there is no items in the second group
+    await expect(page.locator('[data-element="group-aTheBlockListGroupTheSecond"] >> .umb-block-list__block--view')).toHaveCount(0);
+
+    // Copies block list content from the first group
+    await page.locator('[title="Copy"]').click();
+    await expect(page.locator('.alert-success', {hasText: 'Copied to clipboard'})).toBeVisible();
+    // Pastes into the second group
+    await page.locator('[title="Clipboard"]').nth(1).click();
+    await page.locator('umb-block-card', {hasText: elementName}).click();
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.saveAndPublish));
+    await expect(page.locator('.alert-success', {hasText: 'Content Published'})).toBeVisible();
+
+    // Assert
+    await expect(page.locator('[data-element="group-aBlockListGroup"] >> .umb-block-list__block--view')).toHaveCount(1);
+    await expect(page.locator('[data-element="group-aTheBlockListGroupTheSecond"] >> .umb-block-list__block--view')).toHaveCount(1);
+    await page.locator('[data-element="group-aTheBlockListGroupTheSecond"] >> .umb-block-list__block--view').click();
+    await expect(page.locator('[id="sub-view-0"] >> [name="textbox"]')).toHaveValue('aliasTest');
```

Comment:
There is also a new `getDataElementByElementName` we could use, and maybe chain instead here 👍

Reply:
We can't chain with the `getDataElementByElementName`